.github/workflows/release.yaml

```diff
@@ -0,0 +1,247 @@
+      options: -u root
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          ref: ${{ github.ref_name }}
+
+      - name: Install AWS CLI
+        run: apt install awscli -y
+
+      - name: aws creds
+        uses: aws-actions/configure-aws-credentials@v4
+        id: creds
+        with:
+          aws-region: "eu-west-1"
+          role-to-assume: ${{ env.AWS_ROLE_TO_ASSUME }}
+          role-session-name: ReleaseDesktopWalletSession
+          mask-aws-account-id: false
+
+      - name: check if executable exist
+        run: |
+          set +e
+          TARGET_NET=${{env.TARGET_NET}}
+
+          # Prepare filenames
+          if [ $TARGET_NET = "mainnet" ]; then
+             FILENAME="concordium-desktop-wallet-${{ needs.login-aws.outputs.VERSION }}"
+          else
+             FILENAME="concordium-desktop-wallet-${TARGET_NET}-${{ needs.login-aws.outputs.VERSION }}"
+          fi
+
+          FILENAME_LATEST_LINUX="latest-linux.yml"
+
+          OUT_FILENAME="${{ needs.login-aws.outputs.VERSION }}/${TARGET_NET}/${FILENAME}"
+          OUT_LATEST_LINUX="${{ needs.login-aws.outputs.VERSION }}/${TARGET_NET}/latest-linux.yml"
+          aws s3 ls ${{env.S3_BUCKET}}/${OUT_FILENAME}.deb
+          EC=$?
+          if [ $EC -eq 0 ]; then
+            echo ".deb file already exist"
+          elif [ $EC -ne 1 ]; then
+            echo "Error: $EC"
+          fi
+          echo "EXIT_CODE=$EC" >> $GITHUB_ENV
+          echo "OUT_FILENAME=$OUT_FILENAME" >> $GITHUB_ENV
+          echo "FILENAME=$FILENAME" >> $GITHUB_ENV
+          echo "OUT_YML_FILENAME=$OUT_LATEST_LINUX" >> $GITHUB_ENV
+
+
+      - name: Install dependencies and build
+        if: ${{ env.EXIT_CODE == 1 }}
+        run: |
+          rustup default stable
+          node --version
+          npm --version
+          yarn --version
+          python --version
+          rustup show
+          wasm-pack --version
+
+          yarn
+          yarn package
+
+      - name: publish
+        if: ${{ env.EXIT_CODE == 1 }}
+        run: |
+          set +e
+
+          # Push to s3
+          aws s3 cp "release/${{ env.FILENAME }}.deb" "${{env.S3_BUCKET}}/${{ env.OUT_FILENAME }}.deb" --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
+          aws s3 cp "release/${{ env.FILENAME }}.rpm" "${{env.S3_BUCKET}}/${{ env.OUT_FILENAME }}.rpm" --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
+          aws s3 cp "release/${{ env.FILENAME }}.AppImage" "${{env.S3_BUCKET}}/${{ env.OUT_FILENAME }}.rpm" --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
+          aws s3 cp "release/latest-linux.yml" "${{env.S3_BUCKET}}/${{ env.OUT_YML_FILENAME }}" --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
+
+  #using windows-2019 to be able to use visual studio 2019, newer versions do not support older versions of node
```

Comment:
Why do we need to support older versions of the node. Could you write which version of the node causes us to go so low?

I mean if we are protocolversion < 7 why is it even relevant that the wallet cannot communicate with it?

Reply:
This is node.js and not the concordium node, we are using node.js version 16, but there is an error, which is not present in version 14, which is why we are using node.js version 14 both here and in jenkins before, but node.js is not supported by visual studio version 2022